bundles/action/org.openhab.action.pushover/src/main/java/org/openhab/action/pushover/internal/Pushover.java

```diff
@@ -290,6 +312,8 @@ public static boolean pushover(@ParamDoc(name = "apiKey", text = "Your applicati
 
             if (!StringUtils.isEmpty(sound)) {
                 addEncodedParameter(data, MESSAGE_KEY_SOUND, sound);
+            } else if (StringUtils.isEmpty(sound) && !StringUtils.isEmpty(defaultSound)) {
```

Comment:
The part `StringUtils.isEmpty(sound) && ` is unneeded and would read better if it were removed.